js/stateless.js/src/rpc-interface.ts

```diff
@@ -504,7 +504,9 @@ export interface CompressionApiInterface {
 
     getCompressedAccountsByOwner(
         owner: PublicKey,
-        config?: GetCompressedAccountsByOwnerConfig,
+        config?:
+            | Omit<GetCompressedAccountsByOwnerConfig, 'cursor' | 'limit'>
+            | undefined,
```

Comment:
can remove `undefined`, `?` already takes care of it

why exclude `cursor` and `limit` from the param?